src/frontend/components/SøknadsSteg/EøsSteg/Søker/useEøsForSøker.tsx

```diff
@@ -1,27 +1,71 @@
-import { FeltState, ISkjema, ok, useFelt, useSkjema } from '@navikt/familie-skjema';
+import React from 'react';
 
+import { ESvar } from '@navikt/familie-form-elements';
+import { feil, ISkjema, ok, useSkjema } from '@navikt/familie-skjema';
+
+import { useApp } from '../../../../context/AppContext';
+import { useFeatureToggles } from '../../../../context/FeatureToggleContext';
+import useJaNeiSpmFelt from '../../../../hooks/useJaNeiSpmFelt';
+import { usePerioder } from '../../../../hooks/usePerioder';
+import { IArbeidsperiode } from '../../../../typer/perioder';
+import { ISøker } from '../../../../typer/person';
 import { IEøsForSøkerFeltTyper } from '../../../../typer/skjema';
+import { arbeidsperiodeFeilmelding } from '../../../Felleskomponenter/Arbeidsperiode/arbeidsperiodeSpråkUtils';
+import SpråkTekst from '../../../Felleskomponenter/SpråkTekst/SpråkTekst';
 
 export const useEøsForSøker = (): {
     skjema: ISkjema<IEøsForSøkerFeltTyper, string>;
     validerFelterOgVisFeilmelding: () => boolean;
     valideringErOk: () => boolean;
     oppdaterSøknad: () => void;
+    leggTilArbeidsperiode: (periode: IArbeidsperiode) => void;
+    fjernArbeidsperiode: (periode: IArbeidsperiode) => void;
 } => {
-    const placeholderForFeltSomKommer = useFelt<string>({
-        feltId: 'todo',
-        verdi: '',
-        valideringsfunksjon: (felt: FeltState<string>) => {
-            return ok(felt);
-        },
+    const { søknad, settSøknad } = useApp();
+    const søker = søknad.søker;
+    const { toggles } = useFeatureToggles();
+
+    const arbeiderINorge = useJaNeiSpmFelt({
+        søknadsfelt: søker.arbeiderINorge,
+        feilmeldingSpråkId: 'eøs-om-deg.arbeidsperioderinorge.feilmelding',
```

Comment:
Her må vi og ta høyde for enke, men kan kanskje slenge det på den andre enke-bug oppgaven

Reply:
Ingen enke her..

Reply:
Tenker det som omhandler enke kanskje kan tas når man legger det til for andre forelder